tests/InputFiles/ExportCoordinates/Input1D.yaml

```diff
@@ -0,0 +1,18 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: ExportCoordinates1D
+# Check: execute
+# ExpectedOutput:
+#   Coordinates0.h5
+
+DomainCreator:
+  Interval:
+    LowerBound: [0]
+    UpperBound: [1]
+    IsPeriodicIn: [false]
+    InitialRefinement: [1]
+    InitialGridPoints: [3]
+
+VolumeFileName: "Coordinates"
+ReductionFileName: "Reductions"
```

Comment:
If there are no reductions being done, does this need to be specified?

Reply:
It does, but I would be good to remove this requirement #1425